### PR TITLE
website/docs: Balok pr for User docs

### PR DESCRIPTION
**Pull request by tanberry**
This is an expansion of @balok's original PR in the hackathon. Full credit to Balok; I just edited and then finished writing the invitations.md topic. :-)

### CODE REVIEW

File: website/docs/user-group/user/invitations.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+:::info
+
+You can also create a policy to see if the invitation was ever used.
+
+:::
+
+## Create an invitation
+
+Invitations are yet another way to connect Flows, Stages, and Prompts in order to create a speficic user task or worflow.
```
Comment (kensternberg-authentik) [inline website/docs/user-group/user/invitations.md:18]: s/speficic/specific/

"Mike, this [encyclopedia is so old](https://mst3k.fandom.com/wiki/MST3K_820_-_Space_Mutiny#:~:text=Opening%3A%20Crow%20and%20Tom%20think%20the%20encyclopedias%20that%20Mike%20has%20are%20a%20little%20dated.) Congress is spelled with an 'f'! What is this, congreffs"?

File: website/docs/user-group/user/user_basic_operations.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+---
+title: Manage users
+---
+
+The following topics are for the basic management of users: how to create, modify, delete or deactivate users, and using a recovery email.
+
+### Create a user
+
+> If you want to automate user creation, you can do that either by [invitations](./invitations.md), [user write stage](../../flow/stages/user_write), or [using the API](/developer-docs/api/browser).
+
```
Comment (kensternberg-authentik) [inline website/docs/user-group/user/user_basic_operations.md:10]: > If you can automate creating new users by:
>
> - [Invitations](./invitations.md): Invite your users to create their own accounts
> - [Using the API](./developer-docs/api/browser): Create user accounts through automated scripting

Looking at the contents of `/stages/user_write` in the API (I couldn’t find `/flow/stages/user_write` in the API, and the URL `http://localhost:9000/if/admin/#/flow/stages/user_write` in the WebUI gives me a 404), this looks much more like “how I define the user_write stage for a new user flow,” and not “how I define a new user and use the Flow to put them into the system.”

```Javascript
{
    name: string
    flow_set: [{
        name: string
        slug: string
        title: string
        designation: enum
        policy_engine_mode: enum
        compatibility_mode: boolean
        layout: enum
        denied_action: enum
    }]
    user_creation_mode: enum
    create_users_as_inactive: boolean
    user_path_template: string
}
```

File: website/docs/user-group/user/user_basic_operations.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+2. Review the changes and click **Update**.
+
+The active sessions are revoked and the authentication of the user blocked. You can reactivate the account by following the same procedure.
+
+#### To delete a user:
+
+:::caution
+This deletion is not reversible, so be sure you do not need to recover any identity data of the user.
+Otherwise, you can deactivate the account.
```
Comment (kensternberg-authentik) [inline website/docs/user-group/user/user_basic_operations.md:110]: "You may instead deactivate the account to preserve identity data."

File: website/docs/user-group/user/user_basic_operations.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+If the user does not receive the email, check if the mail server parameters [are properly configured](../../troubleshooting/emails).
+
+### Reset the password for the user
+
+As an Admin, you can simply reset the passwrod for the user.
+
+1. In the Admin interface, navigate to **Directory > Users** to display all users.
+
+2. Either click the name of the user to display the full User details page, or click the chevron beside their name to expand the toptions.
+
```
Comment (kensternberg-authentik) [inline website/docs/user-group/user/user_basic_operations.md:93]: "... chevron (the **<span style="font-size: bigger">›</span>** symbol) ..."

File: website/docs/user-group/user/invitations.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+In the Admin UI, navigate to **Directory --> Invitations**, and then click **Create** to open the **Create Invitation** modal. Define the following fields:
+
+-   **Name**: provide a name for your invitation object.
+-   **Expires**: select a date for when you want the invitation to expire.
+-   **Flow**: in the drop-down menu, select the **default-enrollment-flow** Flow.
+-   **Attributes**: (_optional_) enter optional key/value pairs here, to pre-define any information about the user that you will invite to enroll. The \_**\_add info here about the `context['prompt_data']` variable in an expression policy**\_  ***Also link to docs about variables and context flows***
+
+![Create an invitation modal box](./create_invite.png)
+
```
[the remark this answers is not in the document]
Comment (tanberry) [reply website/docs/user-group/user/invitations.md:46]: You're not missing anything. I'll fix the double "is" sentences. 

And... I asked Jens this same question, about why we don't supply the `default-enrollment-flow`... apparently we have so many example flows that it wouldn't be feasible to supply them all out of the box. But yeah, would be easier if it were just there.

File: website/docs/user-group/user/user_basic_operations.md (hunk 2 above)
Comment (tanberry) [reply website/docs/user-group/user/user_basic_operations.md:10]: In a future PR I will attempt to clarify the use of the `user_write` stage. I just want to get these basic CRUD procedurals out now.

File: website/docs/user-group/user/user_attributes.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: User
+title: User attributes
```
Comment (BeryJu) [inline website/docs/user-group/user/user_attributes.md:2]: Probably don't want to call this page "User attributes" when we talk about properties and attributes

**Review by BeryJu — APPROVED** [review]

**Pull request merged**